src/cmd/commit.ts

```diff
@@ -63,6 +63,33 @@ export const commit = async (): Promise<void> => {
   await commitAndPush(values, branch)
 }
 
+export const cloneOtomiChartsInGitea = async (): Promise<void> => {
+  const d = terminal(`cmd:${cmdName}:gitea-otomi-charts`)
+  d.info('Cloning otomi-charts in Gitea')
+  const values = (await hfValues()) as Record<string, any>
+  const { email, username, password } = getRepo(values)
+  const workDir = '/tmp/otomi-charts'
+  const otomiChartsUrl = 'https://github.com/redkubes/otomi-charts.git'
```

Comment:
Let's make it configurable through env with the default to `https://github.com/redkubes/otomi-charts.git`